### PR TITLE
Miscaleneous wiki, script updates

### PR DESCRIPTION
**Pull request by Meister1593**
Added more mentions for linux on troubleshooting page about encoder
Updated some old wiki entries that seemed to be unnecessary or incorrect
Updated link to new ADBForwader
Updated flatpak audio script to newest version
Reformatted all changed wiki md files

### CODE REVIEW

**Review by zarik5 — APPROVED** [review]
That's a lot of changes, but all looks good :)

File: alvr/xtask/flatpak/audio-flatpak-setup.sh [hunk 1]
```diff
@@ -19,52 +31,52 @@ function get_alvr_playback_source_id() {
...
   setup_audio
+  setup_mic
   ;;
 disconnect)
-  unload_mic
-  unload_sink
+  unload_modules
   ;;
-esac
+esac
```
Comment (Vixea) [inline alvr/xtask/flatpak/audio-flatpak-setup.sh:82]: tiny nit but this should follow LF, not CRLF

File: wiki/ALVR-client-and-streamer-on-separate-networks.md [hunk 2]
```diff
@@ -106,15 +116,15 @@ ALVR version Experimental v7 or newer is recommended for this configuration.
...
-1. Oculus Quest must be connected to computer and command-line `adb shell am startservice -n "com.polygraphene.alvr/.ChangeSettings" --es "targetServers" "10.10.10.10"` must be run in Command Prompt to specify IP address of ALVR streamer (`10.10.10.10` must be substituted with IP address of ALVR streamer; the long line is a single command-line).
-1. Next time when ALVR client will be started it should try to connect to the specified ALVR streamer. ALVR streamer should display the client in _Server_ tab (the same way local-network clients are displayed).
 
+1. UDP ports 9943 and 9944 of ALVR streamer must be accessible from Oculus Quest device (i.e. firewall openings must be made to allow Oculus Quest to connect to ALVR streamer UDP ports 9943 and 9944). 
+2. Oculus Quest must be connected to computer and command-line `adb shell am startservice -n "com.polygraphene.alvr/.ChangeSettings" --es "targetServers" "10.10.10.10"` must be run in Command Prompt to specify IP address of ALVR streamer (`10.10.10.10` must be substituted with IP address of ALVR streamer; the long line is a single command-line).
```
Comment (Vixea) [inline wiki/ALVR-client-and-streamer-on-separate-networks.md:121]: Is this correct @zarik5 ?
Should it not be com.alvr-org.alvr or whatever namespace we use now or should this just be removed

**Review by Vixea — CHANGES_REQUESTED** [review]

File: wiki/ALVR-client-and-streamer-on-separate-networks.md (hunk 2 above)
Comment (zarik5) [reply wiki/ALVR-client-and-streamer-on-separate-networks.md:121]: Oh, the client didn't support this for a long time lol. I think this was removed inv12

Comment (zarik5) [reply wiki/ALVR-client-and-streamer-on-separate-networks.md:121]: I think we could remove the whole section `# ALVR v11 and Below`

Comment (zarik5) [reply wiki/ALVR-client-and-streamer-on-separate-networks.md:121]: I think we could remove the whole section # ALVR v11 and Below

File: alvr/xtask/flatpak/audio-flatpak-setup.sh (hunk 1 above)
Comment (Meister1593) [reply alvr/xtask/flatpak/audio-flatpak-setup.sh:82]: it was LF though, just no line after it

File: wiki/ALVR-client-and-streamer-on-separate-networks.md (hunk 2 above)
Comment (Meister1593) [reply wiki/ALVR-client-and-streamer-on-separate-networks.md]: Removed whole section

File: alvr/xtask/flatpak/audio-flatpak-setup.sh (hunk 1 above)
Comment (Meister1593) [reply alvr/xtask/flatpak/audio-flatpak-setup.sh]: added line after it

**Review by Vixea — APPROVED** [review]

**Pull request merged**